packages/components/src/index.ts

```diff
@@ -61,7 +61,8 @@ export {
 	CompositeGroup as __unstableCompositeGroup,
 	CompositeItem as __unstableCompositeItem,
 	useCompositeState as __unstableUseCompositeState,
-} from './composite';
+} from './composite/legacy';
```

Comment:
How much are those used? Can we deprecate them immediately?

Reply:
We will, in a follow-up PR